### PR TITLE
Region biasing or optional arguments

### PR DESCRIPTION
**Pull request by FrancisMurillo**
This adds an optional second keyword list to each API methods to allow extra params to be passed in the url. In particular, this resolves the #2 to allow for region biasing.

### CODE REVIEW

File: lib/google_geocoding_api.ex [hunk 1]
```diff
@@ -22,54 +22,56 @@ defmodule GoogleGeocodingApi do
...
+    result = all_info(address, opts)
     if result, do: List.first(result["results"])["types"]
   end
 
-  defp make_request(address) do
-    params = %{address: address}
-    if key, do: params = Map.put(params, :key, key)
+  defp make_request(address, opts \\ []) do
+    params = %{region: Keyword.get(opts, :region, "")}
+    |> Enum.into(%{address: address})
```
Comment (smythey21) [inline lib/google_geocoding_api.ex]: Would this be simpler?

```
params = %{
  address: address, 
  region: Keyword.get(opts, :region, "")
}
```

File: test/support/api_response_tests.ex [hunk 2]
```diff
@@ -62,6 +62,14 @@ defmodule ApiResponseTests do
           assert response == unquote(success_data)
         end
       end
+
+      test "it adds options to URL if it exists" do
+        Application.put_env(:google_geocoding_api, :region, "XYZ")
```
Comment (smythey21) [inline test/support/api_response_tests.ex]: We shouldn't need the env variable right?

File: lib/google_geocoding_api.ex [hunk 3]
```diff
@@ -3,8 +3,8 @@ defmodule GoogleGeocodingApi do
   Provides functions to interact with Google Geocoding API.
   """
 
-  def all_info(address) do
-    result = Poison.decode!(make_request(address).body)
+  def all_info(address, opts \\ []) do
```
Comment (smythey21) [inline lib/google_geocoding_api.ex:6]: 👌 nice, makes it easy to add other optional arguments down the line. Good thinking

**Review by smythey21 — CHANGES_REQUESTED** [review]
Looks awesome, thanks for contributing! Just a few things from me. I'll try to get this out as a `0.1.1` release ASAP!

File: test/support/api_response_tests.ex (hunk 2 above)
Comment (FrancisMurillo) [reply test/support/api_response_tests.ex]: Oops, forgot to remove that part.

File: lib/google_geocoding_api.ex (hunk 1 above)
Comment (FrancisMurillo) [reply lib/google_geocoding_api.ex]: No problem with this bit.

**Review by smythey21 — APPROVED** [review]
Awesome! I'll try to get this released later tonight 🎉

**Pull request merged**